virttest/qemu_devices/qcontainer.py

```diff
@@ -2739,6 +2762,10 @@ def pcic_by_params(self, name, params, parent_bus=None):
                                 parent_bus=parent_bus,
                                 child_bus=bus)
 
+    def throttle_group_define_by_params(self, group_params, name):
+        props = json.loads(group_params.get("throttle_group_parameters", "{}"))
```

Comment:
Just a suggestion, is there a way to introduce these new parameters(throttle_group_parameters/image_throttle_group...)? e.g. some comments and examples, maybe in shared/base.cfg?

Reply:
The suggestion will be implemented in tq-qemu demo "Add throttle utils and test autotest/tp-qemu#2346"

Reply:
Suggest you could add a unittest to test `class ThrottleGroupManager(object)` in `./avocado-vt/selftests/unit`instead of `qemu/tests/cfg/throttle_utils_test.cfg `.
The case `throttle_utils_test.cfg` should match manual cases according to our rule. If you wanna add it, better to add the corresponding manual cases so that other contributors could review it smoothly.